### DISCUSSION TITLE
[Announcement] Robin Malfait is now a part of Tailwind Labs!

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#2153] [Show and tell]
Back in May we published [our first job posting](https://jobs.tailwindui.com/full-stack-developer) to help us find a full-stack developer to join our team.

After receiving almost 900 applications and interviewing dozens of talented people, we're excited to finally share that [Robin Malfait](https://twitter.com/malfaitrobin) accepted our offer for the position and is officially part of the Tailwind Labs team as of today!

Read the announcement on our blog to learn more about Robin and the process we used to find someone for the role:

[Read "From Nine Hundred to One: How We Hired Robin Malfait"](https://blog.tailwindcss.com/from-900-to-1-how-we-hired-robin-malfait)

Welcome to the team man! 🍾

**Comment by hsmess** [comment]
Wow, this is crazy! That's a massive commitment to hiring. I hope it works out!
Out of interest, did you pay the 10 developers for their time in completing the 4-8 hour challenge?
If not, why not?

Also, if he's OK with it, could you share Robin's code for the challenge? Always interested to see what people would have produced and why it was good.

**Reply by RobinMalfait** [reply]
The code is on my GitHub 👀

**Reply by jvlobo** [reply]
I was also curious to know if the challenge was paid and I came here to ask, but I saw that was already asked, without an answer tho 😅
Is it something you could disclose @adamwathan ??

**Comment by marcelfahle** [comment]
I love this! I'm not super experienced with job applications but all I hear are various degrees of bad. What you guys did feels quite the opposite and very thoughtful. Also, thanks for providing links to the application and demo project, I thought that very interesting. Congrats Robin and congrats Team! I'm sure you'll guys create wonderful things together! 🚀

**Reply by RobinMalfait** [reply]
Thank you!

**Comment by mmikhan** [comment]
I really liked the pairing session idea where you choose something that neither of you did before. Bravo 👏

When I was reading the _pairing session_ section in the blog post, I was thinking _they could have made a date picker_ and reading a bit further, you actually did it 😀

**Comment by Macxim** [comment]
Nice post, I really appreciate the level of transparency and the way you described the whole process.
Congratulations Robin and congrats Tailwind Labs! 🎩

**Reply by RobinMalfait** [reply]
Thanks!

**Comment by heytulsiprasad** [comment]
Wow, congratulations Robin! I just caught up on the blog and I couldn't resist myself to reply after reading a post with such great clarity into the hiring and interviewing process in general. As I've never worked with a brand/company before, I haven't got this much of clarity from someone before and I thank Adam for writing this. 😄

Also I read Robin's really beautiful cover for the posting as well, which was more insightful into how developers work in real world projects along side contribute to the open source community. That also gave me ideas to consider while applying to my very first job post. (sometime soon). 

Thanks everyone for putting out all the content! Love tailwind and looking forward to more awesome features ❤️ 

**Edit**: Can I subscribe somewhere for getting blogs from Tailwind in my mail when they're posted?

**Reply by RobinMalfait** [reply]
Thank you very much!

**Comment by bitwombat** [comment]
For a 1-in-10 chance of getting a job, a 4-8 hour take-home project means up to 40-80 hours of "free" work for a typical candidate, if those odds hold up generally. Something I wish all prospective employers would consider.

Also, I'm actually surprised you didn't hire Robin right off the bat based on his exemplary community reputation, resume, and interview. Extremely low odds he wasn't going to impress through the rest of the hoops.

**Comment by eagostini** [comment]
Just finished reading the [From Nine Hundred to One: How We Hired Robin Malfait](https://blog.tailwindcss.com/from-900-to-1-how-we-hired-robin-malfait) article and I got so inspired!

Reading about the entire process you guys went through to hire the new developer was a huge learning for me and I could somehow feel the pressure as if I was part of it. Congratulations @RobinMalfait and the **Tailwind** team! 🎉

Just wondering now, which path would you choose next time you decide to hire someone, @adamwathan. Would you start from the top fifty last applicants? Would you consider doing it all over again based on a new necessity? Just curious.

**Reply by RobinMalfait** [reply]
Thank you!

**Comment by SanthoshRaju91** [comment]
I really enjoyed reading the blog post, you and your team were so mature in your hiring process. Congratulations @RobinMalfait and Tailwind team.

Just curious what is that "He capitalized the "H" in "GitHub"", is it a metaphor ??

**Reply by RobinMalfait** [reply]
Thanks @SanthoshRaju91!

A lot of people type `Github` instead of `GitHub`.

**Comment by peteraiello** [comment]
Congrats to Robin. After reading the article I was inspired to try the technical test as a change from practising with to-do list apps. Article [here](https://medium.com/@peter_aiello/practicing-react-hooks-using-the-tailwindcss-ui-take-home-test-89ce27ef34c9)

**Comment by quantuminformation** [comment]
I would not go through such an interview process unless I was paid for my time.
